dist/index.js

```diff
@@ -92,24 +93,25 @@ const to = (svg) => {
 };
 const toPng = (svg) => {
     return async (options) => {
-        return convertSvg(svg, Object.assign({}, constants_1.defaultPngShorthandOptions, options));
+        return convertSvg(svg, Object.assign(Object.assign({}, constants_1.defaultPngShorthandOptions), options));
```

Comment:
I think this has been added automatically when updating typescript, I should check it...